agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -332,13 +343,10 @@ private static SdkTracerProviderBuilder configureTracing(
     }
 
     String tracesExporter = config.getString("otel.traces.exporter");
-    if ("none".equals(tracesExporter)) { // "none" is the default set in AiConfigPropertySource
```

Comment:
why "none" and other exporters default were "otel"?

Reply:
"otlp" is the default, but we need to set it to "none" so that we can configure our own exporter later on